utils/feltService.ts

```
@@ -36,3 +36,16 @@ export function gweiToEth(gwei: string): string {
 
   return ethBigInt.toString();
 }
+
+export function tokenToDecimal(tokenValue: string, decimals: number) {
+  if (!tokenValue || isNaN(Number(tokenValue)) || isNaN(Number(decimals)) || decimals < 0) {
```

Biome

[error] isNaN is unsafe. It attempts a type coercion. Use Number.isNaN instead.

See the MDN documentation for more details.
Unsafe fix: Use Number.isNaN instead.


(lint/suspicious/noGlobalIsNan)